libraries/shared/repository/repository_test.go

```diff
@@ -153,10 +149,11 @@ var _ = Describe("Repository", func() {
 		})
 
 		AfterEach(func() {
-			test_config.CleanCheckedHeadersTable(db, getExpectedColumnNames())
+			//test_config.CleanCheckedHeadersTable(db, getExpectedColumnNames())
```

Comment:
Do we want to remove this `AfterEach` entirely?

Reply:
oh nice catch!  i think I commented this out so I could get some data in my test db - I think it should still hang around.